### PR TITLE
feat(etl): fix date format - EUBFR-77

### PR DESCRIPTION
**Pull request by emeryro**
Fix date format in ETLs

### CODE REVIEW

**Review by kalinchernev — DISMISSED** [review]
I'm afraid changes do not work for me.
I am inforegio user, adding the inforegio data, getting the following errors:
```
{
    "errorMessage": "Invalid time value"
}
```

Also, I noticed we've probably missed timeframe mapping for the inforegio transform:

```
    timeframe: {
      from: new Date(record.start).toISOString(),
      to: new Date(record.start).toISOString(),
    },
```

I guess the `to` should be mapped to `end` field of the record?

**Review by raduchiriac — COMMENTED** [review]
@kalinchernev Not sure I did all the right things but <strike>worked for me</strike>.

EDIT: Confirmed, after deploying I get this too: "Invalid time value"

**Comment by kalinchernev** [comment]
Dates are finally correctly input as dates, thanks @yhuard for noticing the format difference!
I haven't updated `services/ingestion/etl/budg/xls/src/lib/transform.js` though, I don't know where to find test data.

File: services/ingestion/etl/inforegio/json/src/lib/transform.js [hunk 1]
```diff
@@ -2,6 +2,12 @@
  * Transform message (REGIO JSON)
  */
 
+// Takes DD/MM/YYYY to MM/DD/YYYY
+const formatDate = date => {
+  const d = date.split(/\//);
+  return new Date(`${d[1]}/${d[0]}/${d[2]}`).toISOString();
```
Comment (yhuard) [inline services/ingestion/etl/inforegio/json/src/lib/transform.js]: 1. I'd check if `d.length === 3` before
2. Don't use strings with the `Date` constructor (see the notice: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date), it's safer to call `Date(d[2], d[1] - 1, d[0])`

**Review by yhuard — APPROVED** [review]

**Pull request merged**